brjs-sdk/sdk/libs/javascript/br-services/src/br/services/html/BRHtmlResourceService.js

```diff
@@ -2,6 +2,8 @@
 * @module br/services/html/BRHtmlResourceService
 */
 
+var TEMPLATES_LOADED = false;
```

Comment:
I think this should be renamed to `templatesLoaded` since the all upper case convention is used for constants rather than static variables.